cat/CaT-service.yaml

```diff
@@ -4683,6 +4702,15 @@ components:
       description: Indicates if the user sent the message or recieved the message
       readOnly: true
       
+    MessageRead:
+      type: string
+      enum:
+        - READ
+        - UNREAD
+        - ALL
+      description: Indicates if the user has read a RECIEVED message or a recipient has read a sent the message
```

Comment:
`has read a sent the message` - doesn't read quite right?

Reply:
Thanks will fix